scanTopicName: /scan
scanTopicQueueSize: 1
pubTopicName: /cmd_vel
pubTopicQueueSize: 1 
